### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by manedurphy**
Hey @rgagnon2,

Would you mind reviewing the changes I've made here when you have any time? There were just a few tweaks I made to get the code to work locally, including migrating to Mongo Atlas, and removing the script tags of the other services in the HTML.

Thank you!

### CODE REVIEW

File: server/db/index.js [hunk 1]
```diff
@@ -1,21 +1,15 @@
...
-      } else {
-        if(err) reject(err);
-        resolve(data);
-      }
-    });
+    const res = await mongoose.connect(URI, { useNewUrlParser: true, useUnifiedTopology: true});
+    
+    if (!res) throw new Error();
+
+    console.log('Connected to Mongo Atlas');
```
Comment (rgagnon2) [inline server/db/index.js:9]: Nice refactor!!

File: server/dev.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // config env vars
 const path = require('path');
-require('dotenv').config({ path: path.resolve(__dirname, '../', '.env')});
+// require('dotenv').config({ path: path.resolve(__dirname, '../', '.env')});
```
Comment (rgagnon2) [inline server/dev.js:3]: Maybe could delete this since it's just commented out :) Unless you're saving it for any particular reason

**Review by rgagnon2 — APPROVED** [review]
Looks good! Just want to give you a heads up though, I think someone from my FEC team wanted to use Mongo Atlas but eventually found out he couldn't because it was hosted in the cloud. I think the requirement is to eventually install a database on the EC2 instance. Mongo Atlas will work for now since you're just working with the vintage code anyway.

File: server/db/index.js (hunk 1 above)
Comment (manedurphy) [reply server/db/index.js:9]: Yes, I used Atlas to get started on the crud operations more quickly. Latest PR now has Docker setup to build a Mongo container. Thank you  for the feedback!

Comment (manedurphy) [reply server/db/index.js:9]: Also, I gave your write access so let me know if you are now able to merge to master.

**Pull request merged**